.github/workflows/release-metrics.yml

```diff
@@ -0,0 +1,15 @@
+name: Send publish metrics to bigquery
+on:
+  release:
+    types: published
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0.2.0
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_BQ_METRICS_KEY }}
+          export_default_credentials: true
+      - run: echo '${{ github.event }}' | jq -c '.release | {package_name: .name | split(" ")[0], name, tag_name, version: .tag_name | split("-v")[1], html_url, target_commitish, created_at, published_at, draft, prerelease}' > releases.json
```

Comment:
this `jq` stuff lets me massage the `github.event` payload to send in some normalized data. ping me and I can point you to where this lands in gcp and my temporary looker modeling for it.

also, I can't really test this until it's appeared at least once on `main`. so likely I'll need to iterate/bugfix